### ISSUE TITLE
Fix terminology: Replace "Bearer Token authorization" with "Bearer authorization scheme"

### ISSUE THREAD

**Issue by rolfedh** [quarkusio/quarkus#37698]
### Description

In various places, our documentation uses the incorrect terminology.
See 
- https://github.com/quarkusio/quarkus/pull/37534#discussion_r1417284212
- https://docs.google.com/spreadsheets/d/1nWTFPL223TcYFwU-2HWyFI_zSpBlztiO0SZlqmLI4WE/edit#gid=1074577126

cc: @sheilamjones

**Comment by sberyozkin** [comment]
@rolfedh I'm not sure what `Bearer authorization scheme` is. If we talk about HTTP `Authorization` header, that it has a scheme property `Basic`, `Bearer` etc. In terms of the Quarkus authentication, we don't talk about `Bearer token authorization`  but `Bearer token authentication`, even though, as I said in one of the comments, in the OAuth2 world, access tokens about authorizing OAuth2 clients, but it does not matter, it at the Quarkus endpoint level, we verify the token.

I know we use `Bearer token authorization`  and `Bearer token authentication` interchangeably, so if you'd like lets review all the occurrences of these 2 terms and agree in which context one of these terms should be used.

**Comment by sberyozkin** [comment]
@rolfedh You are right, `Bearer Authorization Scheme` is a correct expression as such, but it is like a noun, a mechanism to pass the token within the HTTP Authorization header

**Comment by michalvavrik** [comment]
hey @rolfedh , this issue has been opened for a while, please consider what needs to be done, thanks

**Comment by rolfedh** [comment]
Obsolete.
